art/estimators/classification/blackbox.py

```diff
@@ -23,6 +23,8 @@
 import logging
 from typing import Callable, List, Optional, Union, Tuple, TYPE_CHECKING
 
+from functools import total_ordering
```

Comment:
Could you please move this import before `import logging` to group standard library imports in alphabetical order?